index.js

```diff
@@ -58,7 +59,7 @@ module.exports = (neutrino) => {
     .target('web')
     .context(neutrino.options.root)
     .entry('index')
-      .add(neutrino.options.entry === defaultEntry ? defaultTsEntry : neutrino.options.entry)
+      .add([defaultEntry, indexEntry].includes(neutrino.options.entry) ? defaultTsEntry : neutrino.options.entry)
```

Comment:
I don't understand this change.
Basically even if the neutrino entry is index.js, we are making the default entry as index.ts.
Why?